docs/vnext/README.md

```diff
@@ -0,0 +1,80 @@
+# Napa.js vNext (0.4.x-canary)
+vNext is a version with new worker implementation. In Napa.js vNext, napa worker loads and initializes node.js context, which enables most of node.js APIs and addon support.
+
+## Features in vNext
+- Better Node.js API compatibility. Most of [Node APIs](https://nodejs.org/api/index.html) will be supported by default. See [this](./node-api-compatibility.md) for compatibility tracking.
+- Native addon support. All native addon can be loaded in worker by default, unless it's implemented non-thread-safely.
+- Back compatibility. Napa.js API should not be changed and previous napa.js script should continue to work.
+- Resolved incompatibility issue: [#16658](https://github.com/nodejs/node/issues/16658).
```

Comment:
Personally, we don't need to mention this issue here. If you have to, it is a bit better to link an issue of napajs, instead of node.